test/Conversion/TppToLoops/tpp-to-loops.mlir

```diff
@@ -53,6 +53,22 @@ func.func @relu_to_loops(%arg0: memref<3x3xf32>, %arg1: memref<3x3xf32>) {
 
 // -----
 
+// CHECK: func.func @zero_to_loops(
+// CHECK-SAME:  %[[ARG0:.+]]: memref<3x3xf32>) {
+func.func @zero_to_loops(%arg0: memref<3x3xf32>) {
```

Comment:
Can we please add a test case for the scalar case? Do we also support broadcasting? The PR i have open should handle simple broadcasting till we have a proper tpp op.

Reply:
Actually to support scalar we'd need to also allow scalars in the output as only in-place computation is allowed for tpp.zero.
Supporting that is questionable in the first place so actually there should be no scalar case. I'll remove it.

Reply:
ok perfect, not scalar on the output pls.